code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
+
+		if("spin")
+			spin(20, 1)
```

Comment:
Those who know it's true secret will be able to spin counterclockwise. The masses have to settle for clockwise